content/en/docs/concepts/architecture/nodes.md

```diff
@@ -335,7 +335,7 @@ the same time:
   (default 0.01) per second.
 
 The reason these policies are implemented per availability zone is because one
-availability zone might become partitioned from the master while the others remain
+availability zone might become partitioned from the control plane node(s) while the others remain
```

Comment:
```suggestion
availability zone might become partitioned from the control plane while the others remain
```

You might well run your control plane on nodes, but it's not obligatory.

Reply:
+1 to @sftim 's suggestion

Reply:
Nice! Thanks @sftim and @reylejano !